Bio/SeqFeature.py

```diff
@@ -164,9 +164,11 @@ def __init__(self, location=None, type='', location_operator='',
             self.ref_db = ref_db
 
     def _get_strand(self):
+        """Return the strand where the sequence feature is located (PRIVATE)."""
```

Comment:
Here ``_get_strand`` and ``_set_strand`` are used via the ``property`` line later to define the ``strand`` property which does have a docstring. So in a sense they are documented...

Do you agree something like ``Get function for the strand property (PRIVATE).`` is better? The key point I would want to get across is the link to the ``.strand`` property.

(This same point applies to the other get/set property methods)